packages/zero-client/src/client/custom.test.ts

```diff
@@ -172,17 +166,17 @@ test('custom mutators write to the local store', async () => {
     ownerId: '',
     description: '',
     createdAt: 1743018138477,
-  });
+  }).client;
 
   await z.markQueryAsGot(z.query.issue);
   let issues = await z.query.issue;
   expect(issues[0].title).toEqual('foo');
 
-  await z.mutate.issue.setTitle({id: '1', title: 'bar'});
+  await z.mutate.issue.setTitle({id: '1', title: 'bar'}).client;
```

Comment:
@aboodman - this is going to break people pretty hard as all their `await z.mutate.issue.setTitle({id: '1', title: 'bar'});` will no longer work correctly but there will be no type errors to tell them.